client/core/simnet_trade.go

```diff
@@ -1719,99 +1688,126 @@ func (n *notificationReader) find(ctx context.Context, waitDuration time.Duratio
 	})
 }
 
-func (client *tClient) placeOrder(qty, rate uint64, tifNow bool) (string, error) {
+func (s *simulationTest) placeOrder(client *simulationClient, qty, rate uint64, tifNow bool) (string, error) {
 	dc := client.dc()
-	mkt := dc.marketConfig(testMarket)
+	mkt := dc.marketConfig(s.marketName)
 	if mkt == nil {
-		return "", fmt.Errorf("no %s market found", testMarket)
+		return "", fmt.Errorf("no %s market found", s.marketName)
 	}
 
 	tradeForm := &TradeForm{
 		Host:    dexHost,
-		Base:    testBaseID,
-		Quote:   testQuoteID,
+		Base:    s.base.id,
+		Quote:   s.quote.id,
 		IsLimit: true,
 		Sell:    client.isSeller,
 		Qty:     qty,
 		Rate:    rate,
 		TifNow:  tifNow,
 	}
 
-	precisionStr := fmt.Sprintf("%%.%vf %%s", math.Log10(float64(conversionFactors[testBaseID])))
-	qtyStr := fmt.Sprintf(precisionStr, fmtAmt(qty, testBaseID), testBaseSymbol)
-	precisionStr = fmt.Sprintf("%%.%vf %%s", math.Log10(float64(conversionFactors[testQuoteID])))
-	rateStr := fmt.Sprintf(precisionStr, fmtAmt(rate, testQuoteID), testQuoteSymbol,
-		testBaseSymbol)
+	precisionStr := fmt.Sprintf("%%.%vf %%s", math.Log10(float64(s.base.conversionFactor)))
+	qtyStr := fmt.Sprintf(precisionStr, s.fmtAmt(qty, s.base.id), s.base.symbol)
+	precisionStr = fmt.Sprintf("%%.%vf %%s", math.Log10(float64(s.quote.conversionFactor)))
+	rateStr := fmt.Sprintf(precisionStr, s.fmtAmt(rate, s.quote.id), s.quote.symbol,
+		s.base.symbol)
 
 	ord, err := client.core.Trade(client.appPass, tradeForm)
 	if err != nil {
 		return "", err
 	}
 
-	client.log("placed order %sing %s at %s (%s)", sellString(client.isSeller), qtyStr, rateStr, ord.ID[:8])
+	client.lastOrder = ord.ID
+
+	client.log.Infof("placed order %sing %s at %s (%s)", sellString(client.isSeller), qtyStr, rateStr, ord.ID[:8])
```

Comment:
Must be an old bug but log doesn't look right `2022-05-31 18:24:25.360 [INF] T[1]: placed order selling 20.00000000 dcr at 0.000150000 eth%!(EXTRA string=dcr) (7e764dabd4977779)`